Formula/g/goresym.rb

```diff
@@ -0,0 +1,22 @@
+class Goresym < Formula
+  desc "Go symbol recovery tool"
+  homepage "https://github.com/mandiant/GoReSym"
+  url "https://github.com/mandiant/GoReSym/archive/refs/tags/v2.6.3.tar.gz"
+  sha256 "2f5409cb875e053ad0866b97152d6a6353c05d84db4959021cb88ec2a1e74c1b"
+  license "MIT"
+  head "https://github.com/mandiant/GoReSym.git", branch: "master"
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = %w[
+      -s -w
+    ]
+    system "go", "build", *std_go_args(ldflags: ldflags)
+  end
+
+  test do
+    json_output = JSON.parse(shell_output("#{bin}/goresym '#{bin}/goresym'"))
+    assert_equal json_output["BuildInfo"]["Main"]["Path"], "github.com/mandiant/GoReSym"
+  end
```

Comment:
Note: This test basically uses the `goresym` CLI to parse it's own Golang binary, which will then return json data about it, which it then checks the `["BuildInfo"]["Main"]["Path"]` of to confirm that it is all working properly.

Reply:
👍